docs/core/install/linux-fedora.md

```diff
@@ -12,32 +12,20 @@ ms.date: 01/06/2021
 
 [!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
 
-[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+For more information on installing .NET without a package manager, see one of the following articles:
 
-## Install .NET 5.0
-
-[!INCLUDE [linux-prep-intro-generic](includes/linux-prep-intro-generic.md)]
-
-**Fedora 32**
-
-```bash
-sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
-sudo wget -O /etc/yum.repos.d/microsoft-prod.repo https://packages.microsoft.com/config/fedora/32/prod.repo
-```
+- [Install the .NET SDK or the .NET Runtime with Snap.](../linux-snap.md)
+- [Install the .NET SDK or the .NET Runtime with a script.](../linux-scripted-manual.md#scripted-install)
+- [Install the .NET SDK or the .NET Runtime manually.](../linux-scripted-manual.md#manual-install)
 
-**Fedora 33**
+## Install .NET 5.0
 
-```bash
-sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
-sudo wget -O /etc/yum.repos.d/microsoft-prod.repo https://packages.microsoft.com/config/fedora/33/prod.repo
-```
+The latest version of .NET available in the default package repositories for Fedora is .NET 5.0.
 
 [!INCLUDE [linux-dnf-install-50](includes/linux-install-50-dnf.md)]
 
 ## Install .NET Core 3.1
 
-The latest version of .NET available in the default package repositories for Fedora is .NET Core 3.1.
-
 [!INCLUDE [linux-dnf-install-31](includes/linux-install-31-dnf.md)]
 
 ## Supported distributions
```

Comment:
Fedora 34 and 35 (both in development) also have .NET 5.0 and .NET Core 3.1 packages, if it matters.